Felienne/hedy
# [UI FIX] Improves cheatsheet styling

**Description**
In this PR we clean-up the css styling of the cheatsheet to be more dependent on the Tailwind classes and less on styling attributes. We also add a dropdown arrow to indicate that it's a dropdown menu and make the background-color of the menu a lighter color.

**Fixes #1956**

**How to test**
Navigate to the coding page and notice the slightly different styling of the cheatsheet button. As this is mostly a clean-up task verify that everything is still working correctly.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
